src/mark.ts

```diff
@@ -135,6 +139,11 @@ export interface MarkConfig extends ColorMixins, BaseMarkConfig {
    * - If `null`, all data items are included. In this case, invalid values will be interpreted as zeroes.
    */
   invalid?: 'filter' | Hide | null;
+
+  /**
+   * Default band position for a time unit.
+   */
+  timeUnitBandPosition?: 0 | 0.5;
```

Comment:
Why can this only be two values? Please document.

Reply:
Technically it doesn't have to be either 0 or 0.5.  But in practice, these numbers won't make sense if they are not 0 (beginning of the timeUnit interval or 0.5 (middle of the timeUnit interval). 

I can relax it to number.